### PR TITLE
Refactoring LogicalOperations

### PR DESCRIPTION
**Pull request by antonredko1993**

### CODE REVIEW

File: ElementaryTasks/LogicalOperations.cs [hunk 1]
```diff
@@ -6,107 +6,61 @@ namespace ElementaryTasks
...
-                    maxValue = userInput3;
-                }
-            }
-
-            Console.WriteLine($"Value {maxValue} is the biggest");
+            return maxValue;
         }
 
-        public void EvenNmbers()                   
+        public int CalculateResult(int userInput1, int userInput2)
```
Comment (amid0) [inline ElementaryTasks/LogicalOperations.cs:24]: Name of the method is not meaningful

File: ElementaryTasks/LogicalOperations.cs [hunk 2]
```diff
@@ -6,107 +6,61 @@ namespace ElementaryTasks
...
-                Console.WriteLine("Error! You should enter numerical value");
-                return;
-            }
-
+            string plane;
             if (x > 0 && y > 0)
             {
-                Console.WriteLine($"Coordinates of the point ({x},{y}) are in I plane");
+                plane = "I";
+                return plane;
```
Comment (amid0) [inline ElementaryTasks/LogicalOperations.cs:43]: You should't write return in each if block.
The only one return required at the end of method body.

Comment (amid0) [reply ElementaryTasks/LogicalOperations.cs:43]: Please check all methods

File: ElementaryTasks/LogicalOperations.cs (hunk 1 above)
Comment (antonredko1993) [reply ElementaryTasks/LogicalOperations.cs:24]: Done

File: ElementaryTasks/LogicalOperations.cs (hunk 2 above)
Comment (antonredko1993) [reply ElementaryTasks/LogicalOperations.cs:43]: Done

**Pull request merged**